packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/CommunitySection.tsx

```diff
@@ -16,6 +16,8 @@ import { ExternalLinksModule } from '../external_links_module';
 import { GovernanceSection } from '../governance_section';
 import { CommunitySectionSkeleton } from './CommunitySectionSkeleton';
 import useUserActiveAccount from 'hooks/useUserActiveAccount';
+import { CWDivider } from 'views/components/component_kit/cw_divider';
+import DirectoryMenuItem from '../DirectoryMenuItem';
 
 export const CommunitySection = ({ showSkeleton }) => {
```

Comment:
Github ci => 'showSkeleton' is missing in props validation

Reply:
fixed